### ISSUE TITLE
Auto update box art when manually remapping a game match

### ISSUE THREAD

**Issue by Jmatt110** [Phalcode/gamevault-app#324]
When remapping a game due to a failure to map correctly, the Background image is downloaded correctly, but the box art does not automatically update to the new mapping.

Eg. Gamevault maps a game as Solitaire and downloads box art for it but no background image. Manually fix the mapping via RAWG function to Minesweeper results in the correct image being downloaded for Minesweeper, however the original (incorrect) Solitaire box art is not automatically updated to Minesweeper's.

**Comment by Alfagun74** [comment]
Are you sure? Did you clear your local image cache in the settings and restart gamevault completely?

**Comment by Jmatt110** [comment]
Yes, I have cleared the image cache and restarted the client and server multiple times, box art will not change. I can see in the server logs that it's just grabbing the original box art image URL again when the remap is done, across multiple games.

**Comment by Yelo420** [comment]
I think this is a backend problem. As you said the server boxart searcher seems to always find/choose the same boxart.

**Comment by Alfagun74** [comment]
I will take a look at this thank you

**Comment by Alfagun74** [comment]
Hello @Jmatt110,

It appears you've stumbled upon a bug! The Boxart needs an update, and I've identified the issue. Here's what's happening: during the remapping process, we delete all cached game data, retrieve the new data from RAWG, and then attempt to search for the box art. However, there's a snag in the system – the newly cached data is not being taken into account while searching for the box art.

Rest assured, I'm working on a fix for this issue and will have it resolved as soon as possible.

Thank you for bringing this to our attention!
